power_grid_model_c/power_grid_model_cpp/include/power_grid_model_cpp/basics.hpp

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+#ifndef POWER_GRID_MODEL_CPP_BASICS_HPP
+#define POWER_GRID_MODEL_CPP_BASICS_HPP
+
+#ifdef PGM_DLL_EXPORTS
+#error "You should not define PGM_DLL_EXPORTS when using the C++ headers"
+#endif
+
+#include <exception>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "basics.h"
```

Comment:
here and similarly elsewhere: should be
```suggestion
#include <power_grid_model_c/basics.h>
```

please have a look at #236 to see the explanation

Include order should also be updated according to that same PR

Reply:
Noted. Done.